docs/tutorials/get_started.ipynb

```diff
@@ -0,0 +1,1252 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "21f7a26c",
+   "metadata": {},
+   "source": [
+    "# Get started with Qiskit\n",
+    "\n",
+    "This page is for anyone that understands the basics of quantum computing, and wants to get started creating quantum programs. If you want to learn about quantum computing, check out IBM's [Basics of quantum information course](https://learning.quantum-computing.ibm.com/course/basics-of-quantum-information).\n",
+    "\n",
+    "In this tutorial, you'll create a simple quantum program and run it on a simulator. Before you start, you'll need some way of running Qiskit code. See [Installation](https://qiskit.org/documentation/getting_started.html) to get set up.\n",
+    "\n",
+    "\n",
+    "## Write a quantum program\n",
+    "\n",
+    "Qiskit uses quantum circuits to represent quantum programs."
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "id": "3adb4b21",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    "You can build a circuit by creating a new :class:`~.QuantumCircuit` object and adding instructions in sequence. The :class:`~.QuantumCircuit` constructor has one required argument: the number of qubits. The next argument is the number of classical bits, which is zero if not specified."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5e6b1869",
+   "metadata": {},
+   "source": [
+    "Entanglement is an important property of quantum information, and entanglement experiments often start with a _Bell state:_ A specific two-qubit entangled state. The following code creates a circuit that produces a Bell state, then measures it. For a simple experiment that shows the effects of entanglement, check out the [CHSH game](https://learning.www-dev.quantum-computing.ibm.com/course/algorithms-and-protocols/chsh-game-512).\n",
```

Comment:
This CHSH link is temporary, just so you can see what it'll look like. I'll update the link once we have the page in production.